content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,109 @@
+to an individual expense, a :icon:`fa-paperclip` :guilabel:`(paperclip)` icon appears next to the
+:guilabel:`Customer to Reinvoice` and :guilabel:`Analytic Distribution` columns.
+
+When the report is created, the date range for the expenses appears in the :guilabel:`Expense Report
+Summary` field by default. It is recommended to edit this field with a short summary for each report
+to help keep expenses organized. Enter a description for the expense report, such as `Client Trip
+NYC`, or `Office Supplies for Presentation`, in the :guilabel:`Expense Report Summary` field.
+
+Next, select a :guilabel:`Manager` from the drop-down menu to assign a manager to review the report.
+If needed, update the :guilabel:`Journal` field, using the drop-down menu.
+
+.. image:: expense_reports/expense-report-summary.png
+   :align: center
+   :alt: Enter a short description and select a manager for the report.
+
+If some expenses are missing from the report, they can still be added. Click :guilabel:`Add a line`
+at the bottom of the :guilabel:`Expense` tab. A pop-up window appears with all the available
+expenses that can be added to the report, with a status of :guilabel:`To Submit`. Tick the checkbox
+next to each expense being added, then click :guilabel:`Select`.
+
+The items now appear on the report that was just created. If a new expense needs to be added that
+does **not** appear on the list, click :guilabel:`New` to :ref:`create a new expense <expenses/new>`
+and add it to the report.
+
+.. image:: expense_reports/add-an-expense-line.png
+   :align: center
+   :alt: Add more expenses to the report before submitting.
+
+.. note::
+   Expense reports can be created in one of three places:
+
+   #. Navigate to the main :menuselection:`Expenses` app dashboard (also accessed by going to
+      :menuselection:`Expenses app --> My Expenses --> My Expenses`)
+   #. Navigate to :menuselection:`Expenses app --> My Expenses --> My Reports`
+   #. Navigate to :menuselection:`Expenses app --> Expense Reports`
+
+   In any of these views, click :guilabel:`New` to create a new expense report.
+
+.. _expenses/submit:
+
+Submit an expense report
+========================
+
+When an expense report is completed, the next step is to submit the report to a manager for
+approval. To view all expense reports, navigate to :menuselection:`Expenses app --> My Expenses -->
+My Reports`. Open the specific report from the list of expense reports.
+
+.. note::
+   Reports must be individually submitted, and **cannot** be submitted in batches.
+
+If the list is large, grouping the results by status may be helpful since only reports that have a
+:guilabel:`To Submit` status need to be submitted; reports with an :guilabel:`Approved` or
+:guilabel:`Submitted` status do not.
+
+The :guilabel:`To Submit` expenses are identifiable by the :guilabel:`To Submit` status, and by the
+blue text, while the other expenses text appears in black.
```

Comment:
The first portion of this sentence may fall into the "self-explanatory" column... and, again, I'd double-check the validity of the "blue text/black text" statement.